app/src/main/res/values-de/strings.xml

```diff
@@ -1,65 +1,57 @@
+    <string name="previous">Vorheriger Titel</string>
+    <string name="playpause">Wiedergabe / Pause</string>
     <string name="next">Nächster Titel</string>
 
     <!-- Playlists -->
-    <string name="add_file_to_playlist">Datei zu Playlist hinzufügen</string>
-    <string name="add_folder_to_playlist">Ordner zu Playlist hinzufügen</string>
-    <string name="manage_playlists">Playlisten verwalten</string>
+    <string name="add_file_to_playlist">Datei zur Wiedergabeliste hinzufügen</string>
+    <string name="add_folder_to_playlist">Ordner zur Wiedergabeliste hinzufügen</string>
+    <string name="manage_playlists">Wiedergabelisten verwalten</string>
     <string name="playlists">Playlisten</string>
     <string name="create_playlist">Playlist erstellen</string>
     <string name="rename_playlist">Playlist umbenennen</string>
     <string name="playlist_name_exists">Es gibt bereits eine Playlist mit diesem Namen</string>
-    <string name="remove_playlist">Playlist löschen</string>
-    <string name="remove_playlist_description">Diese Aktion wird nur die Playlist löschen, nicht die eigentlichen Dateien.</string>
-    <string name="remove_playlist_description_placeholder">Diese Aktion wird nur die Playlist \"%s\" löschen, nicht die eigentlichen Dateien.</string>
-    <string name="remove_from_playlist">Playlist löschen</string>
-    <string name="remove_from_playlist_description">Diese Aktion wird nur alle Dateien der Playlist \"%s\" löschen, nicht die eigentlichen Dateien.</string>
+    <string name="remove_playlist">Entferne Wiedergabeliste</string>
+    <string name="remove_playlist_description">Entfernt werden nur die ausgewählten Wiedergabelisten, nicht die eigentlichen Dateien.</string>
+    <string name="remove_playlist_description_placeholder">Entfernt wird nur die Wiedergabeliste \"%s\", nicht die tatsächlichen Dateien.</string>
+    <string name="remove_from_playlist">Von Wiedergabeliste entfernen</string>
+    <string name="remove_from_playlist_description">Es werden nur die Elemente aus der Wiedergabeliste \"%s\" entfernt, die eigentlichen Dateien werden nicht gelöscht.</string>
     <string name="delete_the_files_too">Dateien auch löschen</string>
     <string name="open_playlist">Playlist öffnen</string>
-    <string name="all_songs_cannot_be_deleted">Playlist \'Alle Songs\' kann nicht gelöscht werden</string>
-    <string name="empty_playlist">Die derzeitige Playlist ist leer</string>
+    <string name="all_songs_cannot_be_deleted">Wiedergabeliste \'Alle Lieder\' kann nicht gelöscht werden</string>
+    <string name="empty_playlist">Derzeitige Wiedergabeliste ist leer</string>
     <string name="fetching_songs">Rufe Songs ab…</string>
-    <string name="all_songs">Alle Songs</string>
-    <string name="create_playlist_from_folder">Neue Playlist aus Ordner erstellen</string>
+    <string name="all_songs">Alle Lieder</string>
+    <string name="create_playlist_from_folder">Erstelle neue Wiedergabeliste aus Ordner</string>
```

Comment:
`Erstelle neue Playlist aus Ordner` sounds worse than `Neue Playlist aus Ordner erstellen` so I'd stick with the current version.